block_tracker.sh

```diff
@@ -9,12 +9,14 @@ INSTALL_NAME="block_tracker"
 EXECUTABLE_NAME=${INSTALL_NAME/_/-}						# executable has hypen instead of underscore !!!
 GITHUB_URL="github.com"
 GITHUB_RAW_URL="raw.githubusercontent.com"
+GITHUB_BRANCH="testing"
```

Comment:
I'm not sure this makes sense.. We would have to make sure to set it back to "master" before merging. Besides that I don't really see any real advantages for the user. The testing branch may break at any time and shouldn't be installed instead of the master branch version. For testing purpose it's just fine to download the script to any folder you like and run it from there. You for sure put some thought in your decision and I would like to read about that, maybe I'm wrong.. :)